src/components/button-panel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './button';
+
+const ButtonPanel = () => (
+  <div>
+    <div className="panel-row">
+      <Button name="AC" color="white" />
+      <Button name="+/-" color="white" />
+      <Button name="%" color="white" />
+      <Button name="/" />
+    </div>
+    <div className="panel-row">
+      <Button name="7" color="white" />
+      <Button name="8" color="white" />
+      <Button name="9" color="white" />
+      <Button name="X" />
+    </div>
+    <div className="panel-row">
+      <Button name="4" color="white" />
+      <Button name="5" color="white" />
+      <Button name="6" color="white" />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.